app/components/provider_interface/edit_provider_user_permissions_component.html.erb

```diff
@@ -0,0 +1,38 @@
+<legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+  <h1 class="govuk-fieldset__heading">
+    <span class="govuk-caption-xl"><%= permissions_form.provider_user.full_name %></span>
+    Change permissions: <%= permissions_form.provider.name %>
+  </h1>
+</legend>
+
+<span class="govuk-hint">
+  The user will still be able to view all applications made to courses at this organisation.
+</span>
+
+<%= form_with model: permissions_form,
+              method: :patch,
+              url: provider_interface_provider_user_update_permissions_path(
+                provider_id: permissions_form.provider.id,
+                provider_user_id: permissions_form.provider_user.id,
+              ) do |f| %>
+  <%= f.govuk_error_summary %>
+  <%= f.hidden_field :provider_id %>
+  <%= f.govuk_check_boxes_fieldset :permissions do %>
+    <%= f.govuk_check_box :manage_organisations, true, multiple: false,
+                          label: { text: 'Manage organisation' },
+                          hint_text: 'Change permissions between organisations' %>
+
+    <%= f.govuk_check_box :manage_users, true, multiple: false,
+                          label: { text: 'Manage users' },
+                          hint_text: 'Invite or delete users and set their permissions' %>
+
+    <%= f.govuk_check_box :view_safeguarding_information, true, multiple: false,
+                          label: { text: 'View safeguarding information' },
+                          hint_text: 'View sensitive material about the candidate' %>
+
+    <%= f.govuk_check_box :make_decisions, true, multiple: false,
+                          label: { text: 'Make decisions' },
+                          hint_text: 'Make offers, amend offers and reject applications' %>
```

Comment:
I'm working on this page now. Is it ok if I swap those permissions around so the order is more consistent with the prototype and other pages where we list these?